ckanext/validation/plugin/__init__.py

```diff
@@ -344,3 +347,42 @@ def _get_underlying_file(wrapper):
         return wrapper.stream
     return wrapper.file
 
+
+def _should_remove_unsupported_resource_validation_reports(res_dict):
+    if not t.h.asbool(t.config.get('ckanext.validation.clean_validation_reports', False)):
+        return False
+    has_url_type = True
+    try:
+        has_url_type = t.h.asbool(res_dict.get('url_type'))
```

Comment:
`asbool` will choke on empty strings. We don't need it, we can just use `or not url_type`

Reply:
done